src/DateTimeFormatter/index.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types'
```

Comment:
Jamie added a comment to one of my PRs suggesting separating out an index.js and a Component.js file: https://github.com/ministryofjustice/prisonstaffhub/pull/348#discussion_r279771603